templates/openstack/managed.html

```diff
@@ -381,40 +493,42 @@ <h2>Shape it to your needs</h2>
     <ul class="p-matrix u-clearfix">
       <li class="p-matrix__item">
         <div class="p-matrix__content">
-          <h3 class="p-matrix__title">Hosting partners</h3>
-          <p class="p-matrix__desc"><a href="https://insights.ubuntu.com/2017/04/18/unitas-global-and-canonical-provide-fully-managed-enterprise-openstack/" class="p-link--external">Unitas Global</a>, <a href="https://insights.ubuntu.com/2016/08/22/qts-and-canonical-unveil-private-fully-managed-openstack-cloud/" class="p-link--external">QTS</a> and <a href="https://insights.ubuntu.com/2016/07/05/e-shelter-and-canonical-launch-joint-managed-openstack-private-cloud/" class="p-link--external">eShelter</a> are certified partners, trained to handle the physical BootStack operations.</p>
+          <h3 class="p-matrix__title">Fully managed Kubernetes</h3>
+          <p>Canonical also offers <a href="/kubernetes/managed">managed Kubernetes clusters</a> on bare metal, OpenStack, VMware or public cloud. Combined with our managed OpenStack you have a full cloud and container infrastructure, on prem.</p>
         </div>
       </li>
       <li class="p-matrix__item">
         <div class="p-matrix__content">
-          <h3 class="p-matrix__title">Build services</h3>
-          <p class="p-matrix__desc">We can build your cloud. Get started fast with our OpenStack reference architecture or a custom design to optimise your workloads.</p>
-          <p class="p-matrix__desc"><a href="/openstack/consulting">OpenStack build service&nbsp;&rsaquo;</a></p>
+          <h3 class="p-matrix__title">Cloud build services</h3>
+          <p>Even if you want to operate the cloud from the beginning, we can still help you design and build it.</p>
+          <p>Get started with our <a href="/openstack/consulting">reference architecture</a> to optimise your workloads.</p>
```

Comment:
Get started **fast** with our reference architecture to optimise your workloads.